invenio_records_resources/resources/errors.py

```diff
@@ -70,6 +74,36 @@ def __init__(self, error):
         super().__init__(code=500, description=_("Internal server error"))
 
 
+class HTTPJSONException(_HTTPJSONException):
```

Comment:
this was moved here from invenio-rdm-records as it's required in this module as well